keylime/db/verifier_db.py

```diff
@@ -47,6 +47,7 @@ class VerfierMain(Base):
     ak_tpm = Column(String(500))
     mtls_cert = Column(String(2048), nullable=True)
     attestation_count = Column(Integer)
+    last_received_quote = Column(Float)
```

Comment:
Why is this a float? I thought epoch timestamps were plain integers.

Reply:
In python3, `time.time()` is a `float`

Reply:
I don't have strong feelings either way, but do we ever need subsecond precision for this? Ints are a lot more efficient than floats, but it's not like we're doing any sort of high speed calculation on this field so it's not really going to matter. So I guess whatever is easiest to use is best.